radixdlt-core/radixdlt/src/main/java/com/radixdlt/middleware2/network/ConsensusEventMessage.java

```diff
@@ -136,17 +144,14 @@ public boolean equals(Object o) {
 		if (this == o) {
 			return true;
 		}
-		if (o == null || getClass() != o.getClass()) {
-			return false;
-		}
-		ConsensusEventMessage that = (ConsensusEventMessage) o;
-		return Objects.equals(proposal, that.proposal)
-				&& Objects.equals(vote, that.vote)
-				&& Objects.equals(getTimestamp(), that.getTimestamp());
+
+		return (o instanceof ConsensusEventMessage that)
+			   && Objects.equals(proposal, that.proposal)
+			   && Objects.equals(vote, that.vote);
 	}
 
 	@Override
 	public int hashCode() {
-		return Objects.hash(proposal, vote, getTimestamp());
+		return Objects.hash(proposal, vote);
```

Comment:
Whats the reasoning for removing timestamp? Seems to me without timestamp we can't tell the difference between messages with the same data but sent at different times?

Reply:
- we have no control over timestamp assignment
- we don't store nor compare such events
- in this particular case, we use/process payload rather than container itself

Reply:
But is it related to the task? Maybe it's better to create a new task to investigate it deeper.

Reply:
I agree with @diegomrsantos. We should have big changes like this in a separate PR, especially since it is hard to tell if there are unintended side effects. Also, still not sure this is right way to go anyway.

Reply:
https://radixdlt.atlassian.net/browse/NT-1